### PR TITLE
FAT2-363 EPAO availability on course search

### PR DESCRIPTION
**Pull request by PeterDavidWilliams**

### CODE REVIEW

**Review by dashton82 — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.FindEpao.Domain/Courses/EpaoListItem.cs [hunk 1]
```diff
@@ -9,6 +10,7 @@ public class EpaoListItem
         public string City { get; set; }
         public string Postcode { get; set; }
         public IReadOnlyList<EpaoDeliveryArea> DeliveryAreas { get; set; }
+        public DateTime? EffectiveFrom { get; set; }
```
Comment (dashton82) [inline src/SFA.DAS.FindEpao.Domain/Courses/EpaoListItem.cs:13]: Should it be nullable?

File: src/SFA.DAS.FindEpao.Web/Views/Courses/CourseEpaos.cshtml [hunk 2]
```diff
@@ -32,7 +32,15 @@
...
-                                    <dt class="das-definition-list__title govuk-!-margin-bottom-1">Locations covered</dt><dd class="govuk-!-margin-bottom-1">@epao.Locations</dd>
+                                    @if (epao.EffectiveFrom <= DateTime.Today)
+                                    {
+                                        <dt class="govuk-!-margin-bottom-1">Available to assess</dt>
+                                    }
+                                    @if (epao.EffectiveFrom > DateTime.Today)
+                                    {
+                                        <dt class="govuk-!-margin-bottom-1">Availabe to assess from @epao.EffectiveFrom.Value.ToString("d MMMM yyyy")</dt>
+                                    }
+<dt class="das-definition-list__title govuk-!-margin-bottom-1">Locations covered</dt><dd class="govuk-!-margin-bottom-1">@epao.Locations</dd>
```
[the remark this answers is not in the document]
Comment (PeterDavidWilliams) [reply src/SFA.DAS.FindEpao.Web/Views/Courses/CourseEpaos.cshtml]: fixed. Trolls have been scorned and have fled. For now.

File: src/SFA.DAS.FindEpao.Web/Models/EpaoListItemViewModel.cs [hunk 3]
```diff
@@ -17,12 +17,14 @@ public class EpaoListItemViewModel : EpaoWithLocation
...
             Postcode = epao.Postcode;
+            EffectiveFrom = epao.EffectiveFrom;
         }
 
         public string EpaoId { get; }
         public string Name { get; }
         public string City { get; }
         public string Postcode { get; }
         public string Address => $"{City}, {Postcode}";
+        public DateTime? EffectiveFrom { get; set; }
```
[the remark this answers is not in the document]
Comment (PeterDavidWilliams) [reply src/SFA.DAS.FindEpao.Web/Models/EpaoListItemViewModel.cs]: Change made

File: src/SFA.DAS.FindEpao.Domain/Courses/EpaoListItem.cs (hunk 1 above)
Comment (PeterDavidWilliams) [reply src/SFA.DAS.FindEpao.Domain/Courses/EpaoListItem.cs]: Change made

**Review by dashton82 — APPROVED** [review]

**Pull request merged**